demo/src/scheduleFixedDynamicDispatchTask.js

```diff
@@ -2,75 +2,79 @@ require("@oak-network/api-augment");
 const { rpc, types } = require("@oak-network/types");
 const { ApiPromise, WsProvider, Keyring } = require("@polkadot/api");
 const { waitReady } = require("@polkadot/wasm-crypto");
-const _ = require("lodash");
+const lodash = require("lodash");
 
 const SS58_PREFIX = 51;
 const TRANSFER_AMOUNT = 1000000000;
 const RECEIVER_ADDRESS = "66fhJwYLiK87UDXYDQP9TfYdHpeEyMvQ3MK8Z6GgWAdyyCL3";
 
+const findEvent = (events, section, method) =>
+  events.find((e) => e.event.section === section && e.event.method === method);
+
+const getTaskIdInTaskScheduledEvent = (event) =>
+  Buffer.from(event.event.data.taskId).toString();
+
 const getHourlyRecurringTimestamps = (startTimestamp, numberRecurring) => {
   const secondsInHour = 60 * 60 * 1000;
   const firstEventTimestamp =
     startTimestamp - (startTimestamp % secondsInHour) + secondsInHour;
-  return _.times(
+  return lodash.times(
     numberRecurring,
     (index) => firstEventTimestamp + index * secondsInHour,
   );
 };
 
 const getKeyringPair = async () => {
   await waitReady();
-  if (_.isEmpty(process.env.SENDER_MNEMONIC)) {
+  if (lodash.isEmpty(process.env.SENDER_MNEMONIC)) {
     throw new Error("The SENDER_MNEMONIC environment variable is not set.");
   }
   // Generate sender keyring pair from mnemonic
-  const keyring = new Keyring({ type: "sr25519", ss58Format: SS58_PREFIX });
+  const keyring = new Keyring({ ss58Format: SS58_PREFIX, type: "sr25519" });
   const keyringPair = keyring.addFromMnemonic(process.env.SENDER_MNEMONIC);
   return keyringPair;
 };
 
 const sendExtrinsic = (extrinsic, api, keyringPair) =>
```

Comment:
Extract this function to a utils.js file so all files can reference it, the same as xcm-demo.